### PR TITLE
[ISSUE #4487]The trackType is wrong when the consumer in broadcasting subscription.

### PR DESCRIPTION
**Pull request by zhangjidi2016**
**Make sure set the target branch to `develop`**

## What is the purpose of the change

#4487

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily. Notice, `it would be helpful if you could finish the following 5 checklist(the last one is not necessary)before request the community to review your PR`.

- [x] Make sure there is a [Github issue](https://github.com/apache/rocketmq/issues) filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue. 
- [x] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
- [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [x] Write necessary unit-test(over 80% coverage) to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/rocketmq/tree/master/test).
- [x] Run `mvn -B clean apache-rat:check findbugs:findbugs checkstyle:checkstyle` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.
- [ ] If this contribution is large, please file an [Apache Individual Contributor License Agreement](http://www.apache.org/licenses/#clas).

### CODE REVIEW

**Review by Kvicii — DISMISSED** [review]

File: tools/src/main/java/org/apache/rocketmq/tools/admin/DefaultMQAdminExtImpl.java [hunk 1]
```diff
@@ -320,8 +321,18 @@ public ConsumeStats examineConsumeStats(String consumerGroup,
...
+            try {
+                connection = this.examineConsumerConnectionInfo(consumerGroup);
+            } catch (Exception e) {
+                throw new MQClientException(ResponseCode.CONSUMER_NOT_ONLINE,
+                    "Not found the consumer group consume stats, because return offset table is empty, maybe the consumer not online");
+            }
+
+            if (connection.getMessageModel().equals(MessageModel.BROADCASTING)) {
+                throw new MQClientException(ResponseCode.CONSUME_BROADCASTING,
+                    "Not found the consumer group consume stats, because return offset table is empty, the consumer using in broadcast model");
```
Comment (tsunghanjacktsai) [inline tools/src/main/java/org/apache/rocketmq/tools/admin/DefaultMQAdminExtImpl.java:334]: Hi @zhangjidi2016 ,

Thx for your contribution.

Please rephrase the "the consumer using in broadcast model" into "the consumer is under the broadcast mode". A good exception message would make us more perfect :-)

File: common/src/main/java/org/apache/rocketmq/common/protocol/ResponseCode.java [hunk 2]
```diff
@@ -80,4 +80,6 @@ public class ResponseCode extends RemotingSysResponseCode {
 
     public static final int UPDATE_GLOBAL_WHITE_ADDRS_CONFIG_FAILED = 211;
 
+    public static final int CONSUME_BROADCASTING = 212;
```
[the remark this answers is not in the document]
Comment (zhangjidi2016) [reply common/src/main/java/org/apache/rocketmq/common/protocol/ResponseCode.java:83]: Good suggestion. I'll change it later

**Review by tsunghanjacktsai — APPROVED** [review]
LGTM now 👍

**Review by hzh0425 — APPROVED** [review]

**Comment by Oliverwqcwrw** [comment]
I packed and tested that the trackType was always NOT_CONSUME_YET, but it was SUCCESS in  Consume MessageTrace

**Pull request closed without merging**